pkg/spec/tf_job.go

```diff
@@ -240,6 +253,14 @@ func (c *TfJobSpec) SetDefaults() error {
 			r.setDefaultPSPodTemplateSpec(c.TfImage)
 		}
 	}
+	if c.TerminationPolicy == nil {
+		c.TerminationPolicy = &TerminationPolicySpec{
```

Comment:
Why are you setting  the default policy here and not in 
https://github.com/tensorflow/k8s/blob/master/pkg/spec/tf_job.go#L215?

In validate can we check that termination policy is a valid policy? So right now the only valid policy should be MASTER, index=0

Reply:
It's in the SetDefaults(), I guess you thought it's in Validate because the code collapsed.

Added validation.

Reply:
Thanks. 
Resolved.